### PR TITLE
Update Benchmark data with latest numbers

### PR DESCRIPTION
**Pull request by premkumr**
- Updating Benchmark data & stubs

@netlify /preview/benchmark/

### CODE REVIEW

File: docs/content/preview/benchmark/_index.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ aliases:
   - /preview/benchmark/performance/
 type: indexpage
 ---
+
+YugabyteDB is designed to provide high availability, scalability, and fault tolerance while providing simple interfaces via YSQL and YCQL APIs. However, to assess its true capabilities and to showcase its potential to handle real-world workloads, rigorous benchmarking is essential.
+
+Benchmarking is the process of evaluating the performance and capabilities of a system under specific workloads to gain insights into its scalability, resilience, and overall efficiency. This process involves simulating real-world usage scenarios using standardized workloads to understand how well the system performs, scales, and recovers from failures. It is crucial to understand the ability of YugabyteDB to handle various workloads, such as the TPC-C, YCSB, and Sysbench benchmarks, which represent different aspects of a distributed database's performance.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/_index.md:14]:
```suggestion
Benchmarking is the process of evaluating the performance and capabilities of a system under specific workloads to gain insights into its scalability, resilience, and overall efficiency. This process involves simulating real-world usage scenarios using standardized workloads to understand how well the system performs, scales, and recovers from failures. It is crucial to understand the ability of YugabyteDB to handle various workloads, such as the TPC-C, YCSB, and sysbench benchmarks, which represent different aspects of a distributed database's performance.
```

File: docs/content/preview/benchmark/_index.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ aliases:
...
+
+## TPC-C (Transaction Processing Performance Council - Benchmark C)
+
+[TPC-C](http://www.tpc.org/tpcc/) is a widely recognized benchmark for testing the performance of transactional database systems. It simulates a complex OLTP (Online Transaction Processing) workload that involves a mix of different transactions like order creation, payment processing, and stock level checking. Benchmarking YugabyteDB using TPC-C helps assess its ability to handle a high volume of concurrent transactions and maintain consistency and integrity.
+
+## YCSB (Yahoo Cloud Serving Benchmark)
+
+[YCSB](https://github.com/brianfrankcooper/YCSB/wiki) is designed to evaluate the performance of databases under various read and write workloads, ranging from mostly read-heavy to mostly write-heavy. By using YCSB, you can assess how well YugabyteDB handles different data access patterns and query loads, which is crucial for applications with diverse usage requirements.
+
+## Sysbench
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/_index.md:24]:
```suggestion
## sysbench
```

File: docs/content/preview/benchmark/_index.md [hunk 3]
```diff
@@ -8,6 +8,26 @@ aliases:
...
+
+Benchmarking is the process of evaluating the performance and capabilities of a system under specific workloads to gain insights into its scalability, resilience, and overall efficiency. This process involves simulating real-world usage scenarios using standardized workloads to understand how well the system performs, scales, and recovers from failures. It is crucial to understand the ability of YugabyteDB to handle various workloads, such as the TPC-C, YCSB, and Sysbench benchmarks, which represent different aspects of a distributed database's performance.
+
+## TPC-C (Transaction Processing Performance Council - Benchmark C)
+
+[TPC-C](http://www.tpc.org/tpcc/) is a widely recognized benchmark for testing the performance of transactional database systems. It simulates a complex OLTP (Online Transaction Processing) workload that involves a mix of different transactions like order creation, payment processing, and stock level checking. Benchmarking YugabyteDB using TPC-C helps assess its ability to handle a high volume of concurrent transactions and maintain consistency and integrity.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/_index.md:18]:
```suggestion
[TPC-C](http://www.tpc.org/tpcc/) is a widely recognized benchmark for testing the performance of transactional database systems. It simulates a complex OLTP (Online Transaction Processing) workload that involves a mix of different transactions like order creation, payment processing, and stock level checking. Benchmarking YugabyteDB using TPC-C helps assess its ability to handle a high volume of concurrent transactions, and maintain consistency and integrity.
```

File: docs/content/preview/benchmark/_index.md [hunk 4]
```diff
@@ -8,6 +8,26 @@ aliases:
...
+
+## YCSB (Yahoo Cloud Serving Benchmark)
+
+[YCSB](https://github.com/brianfrankcooper/YCSB/wiki) is designed to evaluate the performance of databases under various read and write workloads, ranging from mostly read-heavy to mostly write-heavy. By using YCSB, you can assess how well YugabyteDB handles different data access patterns and query loads, which is crucial for applications with diverse usage requirements.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/_index.md:22]:
```suggestion
[YCSB](https://github.com/brianfrankcooper/YCSB/wiki) is designed to evaluate the performance of databases under various read and write workloads, ranging from mostly read-heavy to write-heavy. Using YCSB, you can assess how well YugabyteDB handles different data access patterns and query loads, which is crucial for applications with diverse usage requirements.
```

File: docs/content/preview/benchmark/_index.md [hunk 5]
```diff
@@ -8,6 +8,26 @@ aliases:
...
+
+## Sysbench
+
+[Sysbench](https://github.com/akopytov/sysbench) is a versatile benchmarking tool that covers a wide range of database workloads, including CPU, memory, disk I/O, and database operations. It helps measure the system's performance, stability, and scalability under different stress conditions, enabling you to identify potential bottlenecks and weaknesses.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/_index.md:26]:
```suggestion
[sysbench](https://github.com/akopytov/sysbench) is a versatile benchmarking tool that covers a wide range of database workloads, including CPU, memory, disk I/O, and database operations. It helps measure the system's performance, stability, and scalability under different stress conditions, enabling you to identify potential bottlenecks and weaknesses.
```

File: docs/content/preview/benchmark/sysbench-ysql.md [hunk 6]
```diff
@@ -133,13 +133,13 @@ When run on a 3-node cluster with each node on a c5.4xlarge AWS instance (16 cor
...
-| :------- | :-------------------- | :----------- |
-| OLTP_READ_ONLY | 3276 | 39 |
-| OLTP_READ_WRITE | 487 | 265 |
-| OLTP_WRITE_ONLY | 1818 | 70 |
-| OLTP_POINT_SELECT| 95695 | 1.3 |
-| OLTP_INSERT | 6348 | 20.1 |
-| OLTP_UPDATE_INDEX | 4052 | 31 |
-| OLTP_UPDATE_NON_INDEX | 11496 | 11 |
-| OLTP_DELETE | 67499 | 1.9 |
+|       Workload        | Throughput (txns/sec) | Latency (ms) |
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/sysbench-ysql.md:136]: adding the extra spaces to column titles or the entries in each column doesn’t change the table orientation in any way. In short , the table looks the same as before. Not sure why the spaces.

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 7]
```diff
@@ -15,35 +15,47 @@ menu:
     weight: 4
 type: indexpage
 showRightNav: true
+rightNav:
+  hideH3: true
 ---
 
-## Overview
+[TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types and complexity that are either executed online or queued for deferred execution.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:22]:
```suggestion
[TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types, and complexity that are either executed online or queued for deferred execution.
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 8]
```diff
@@ -15,35 +15,47 @@ menu:
...
 
-Follow the steps below to run the [TPC-C workload](https://github.com/yugabyte/tpcc) against YugabyteDB YSQL. [TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types and complexity that are either executed online or queued for deferred execution.
+## Results Overview (v2.18.1)
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:24]:
```suggestion
## Results overview (YugabyteDB v2.18.1)
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 9]
```diff
@@ -15,35 +15,47 @@ menu:
...
 
-### Results at a glance (Amazon Web Services)
+All the nodes in the cluster were located in AWS-West in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for 30 minutes after the loading of the data.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:26]:
```suggestion
All the nodes in the cluster were located in AWS-west in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the database cluster. Each test was run for 30 minutes after loading the data.
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 10]
```diff
@@ -15,35 +15,47 @@ menu:
...
 
-| Warehouses| TPMC | Efficiency (approx) | Cluster Details |
-| :-------- |:---- | :------------------ | :-------------- |
-| 10    | 127      | 98.75%   | 3 nodes of type `c5d.large` (2 vCPUs) |
-| 100   | 1,271.77 | 98.89%   | 3 nodes of type `c5d.4xlarge` (16 vCPUs) |
-| 1,000  | 12563.07 | 97.90%   | 3 nodes of type `c5d.4xlarge` (16 vCPUs) |
-| 10,000 | 125163.2 | 97.35%   | 30 nodes of type `c5d.4xlarge` (16 vCPUs) |
+### Horizontal Scaling
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:28]:
```suggestion
### Horizontal scaling
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 11]
```diff
@@ -15,35 +15,47 @@ menu:
...
 
-All the nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster and was deployed in the same zone as the DB cluster. Each test was run for 30 minutes after the loading of the data.
+This data shows how YugabyteDB scales horizontally - Increased throughput with the same efficiency as the number of nodes in the cluster is increased.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:30]:
```suggestion
The following table shows how YugabyteDB scales horizontally, that is, increased throughput with the same efficiency when the number of nodes in the cluster is increased.
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 12]
```diff
@@ -15,35 +15,47 @@ menu:
...
-| Warehouses | TPMC | Efficiency (approx) | Cluster details |
-| :--------- | :--- | :-----------------: | :-------------- |
-| 50 | 639.4 | 99.44% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
-| 100 | 1,271.37 | 98.86% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
-| 1000 | 12,523.97 | 97.39% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
-| 2000 | 25,407.43 | 98.78% | 3 nodes of type `D16 v3` (16 vCPUs) with `P40` disks |
+### Vertical Scaling
 
-All nodes in the cluster were in the same zone. The benchmark VM was the same type as the nodes in the cluster, and was deployed in the same zone as the DB cluster. Each test was run for 30 minutes after the loading of the data.
+This data shows how YugabyteDB scales vertically - Increased throughput when the power of the machines is increased while keeping the number of nodes in the cluster the same.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:41]:
```suggestion
The following table shows how YugabyteDB scales vertically, that is, increased throughput when the power of the machines is increased while keeping the number of nodes in the cluster the same.
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 13]
```diff
@@ -15,35 +15,47 @@ menu:
...
+
+### 100K Warehouses
+
+| Warehouses |    TPMC    | Efficiency(%) | Nodes | Connections | New Order Latency |       Machine Type (vCPUs)       |
+| ---------: | :--------- | :-----------: | :---: | ----------- | :---------------: | :------------------------ |
+|    100,000 | 1283804.18 |     99.83     |  59   | 1000        |     51.86 ms      | c5d.9xlarge&nbsp;(36) |
+
+## Setup for benchmarking
+
+Follow the steps below to run the [TPC-C workload](https://github.com/yugabyte/tpcc) against YugabyteDB YSQL.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:58]:
```suggestion
Run a [TPC-C workload](https://github.com/yugabyte/tpcc) against YugabyteDB YSQL using the following steps:
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 14]
```diff
@@ -63,9 +75,9 @@ Start your YugabyteDB cluster by following the steps for a [manual deployment](.
 You will need the IP addresses of the nodes in the cluster for the next step.
 {{< /tip>}}
 
-## Configure DB connection parameters (optional)
+### Configure DB connection parameters (optional)
 
-Workload configuration like IP addresses of the nodes, number of warehouses and number of loader threads can be controlled by command line arguments.
+Workload configurations like IP addresses of the nodes, number of warehouses and number of loader threads can be controlled by command line arguments.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:80]:
```suggestion
Workload configurations like IP addresses of the nodes, number of warehouses, and number of loader threads can be controlled by command line arguments.
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 15]
```diff
@@ -75,15 +87,15 @@ Other options like username, password, port, etc. can be changed using the confi
 <password></password>
 ```
 
-## Best practices
+### Other considerations
 
-**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest released version, or you can clone the repository and build from source to get the very latest changes.
+**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest released version, or you can clone the repository and build from the source to get the very latest changes.
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:92]:
```suggestion
**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest [released](https://github.com/yugabyte/tpcc/releases) version, or you can clone the repository and build from the source to get the very latest changes.
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 16]
```diff
@@ -15,35 +15,47 @@ menu:
...
+|       2000 | 25646.4  |     99.78     |   3   | 200         |     54.21 ms      | m6i.2xlarge&nbsp;(8)   |
+|       4000 | 51343.5  |     99.81     |   3   | 400         |     39.46 ms      | m6i.4xlarge&nbsp;(16)  |
+
+### 100K Warehouses
+
+| Warehouses |    TPMC    | Efficiency(%) | Nodes | Connections | New Order Latency |       Machine Type (vCPUs)       |
+| ---------: | :--------- | :-----------: | :---: | ----------- | :---------------: | :------------------------ |
+|    100,000 | 1283804.18 |     99.83     |  59   | 1000        |     51.86 ms      | c5d.9xlarge&nbsp;(36) |
+
+## Setup for benchmarking
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:56]:
```suggestion
## Benchmark setup
```

File: docs/content/preview/benchmark/tpcc-ysql/_index.md [hunk 17]
```diff
@@ -75,15 +87,15 @@ Other options like username, password, port, etc. can be changed using the confi
...
 
-**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest released version, or you can clone the repository and build from source to get the very latest changes.
+**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest released version, or you can clone the repository and build from the source to get the very latest changes.
 
 **Pre-compacting tables:** Pre-compact tables with the [yb-admin](../../admin/yb-admin/) utility's `compact_table` command.
 
 **Warming the database:** Use the `--warmup-time-secs` flag when you call the execute phase of the TPCC benchmark.
 
-## Run the TPC-C benchmark
+## Running TPC-C
```
Comment (aishwarya24) [inline docs/content/preview/benchmark/tpcc-ysql/_index.md:98]:
```suggestion
## Run TPC-C
```

**Review by aishwarya24 — COMMENTED** [review]
Left some suggested changes.

**Review by ddhodge — APPROVED** [review]

**Pull request merged**